fenix/app/src/test/java/org/mozilla/fenix/shopping/ProductAnalysisTestData.kt

```diff
@@ -13,7 +13,7 @@ object ProductAnalysisTestData {
         grade: String? = "A",
         adjustedRating: Double = 4.5,
         needsAnalysis: Boolean = false,
-        lastAnalysisTime: Int = 0,
+        lastAnalysisTime: Float = 0L,
```

Comment:
```suggestion
        lastAnalysisTime: Long = 0L,
```

I believe you're looking for `Long` here?